go-bindings/aggregationinfo.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Chia Network Inc
```

Comment:
This is probably incorrect. IMO we should retain their copyright in addition to ours

Reply:
@PastaPastaPasta I think that specific file is 100% our work though.

Reply:
Yeah Chia doesn't have the Go bindings, they were mentioned there probably because nathan wanted to merge it into their repo but that never happened obviously.

Reply:
Yeah personally I'm not sure if it matters since it wasn't merged in. It could be eventually (but not likely at this point) and is kinda derivative of that project.

I followed @antouhou's lead as he assigned the copyright to Chia for the JS bindings: https://github.com/Chia-Network/bls-signatures/blob/683a1486c1442acfbd679d0b034329abaa7676f3/js-bindings/helpers.cpp#L1